Core/HLE/proAdhoc.h

```diff
@@ -57,6 +57,8 @@
 #include "Core/HLE/sceUtility.h"
 
 #ifdef _WIN32
+#undef min
+#undef max
```

Comment:
Instead of this, see if you can include Common/CommonWindows.h (which sets NOMINMAX which prevents this from being needed) a bit earlier.